src/syncer.py

```diff
@@ -262,9 +265,19 @@ def _addAlias(self, alias, goto, mailcowAliases):
         }, alias)
         pass
 
-    def _addListFilter(self, listAddress, memberAddresses, mailcowFilters):
+    def _addListFilter(self, listAddress, memberAddresses, description, mailcowFilters):
         scriptData = "### Auto-generated mailinglist filter by linuxmuster ###\r\n\r\n"
-        scriptData += "require \"copy\";\r\n\r\n"
+        scriptData += "require \"editheader\";\r\n"
+        scriptData += "require \"copy\";\r\n"
+        scriptData += "require \"variables\";\r\n"
+        scriptData += "set \"addendum\" \""+description+"\";\r\n"
```

Comment:
Why are you using description here instead of cn? Is there a way to alter the description from the webui?